parser_email.py

```diff
@@ -786,7 +786,23 @@ def _add_body_in_email_headers(parsed_mail, file_path, charset, content_type):
             body_content = f.read()
         _debug_print("Reading file data using binary mode")
     # Add body to the last added Email artifact
-    body_content = UnicodeDammit(body_content).unicode_markup.encode('utf-8').decode('utf-8')
+    body_content = UnicodeDammit(body_content).unicode_markup.encode('utf-8').decode('utf-8').replace('\u0000', '')
+
+    _debug_print('Processing email part with content_type: %s' % content_type)
+
+    IMAGE_CONTENT_TYPES = ['image/jpeg', 'image/png']
+
+    if any(t for t in IMAGE_CONTENT_TYPES if t in content_type):
+        _debug_print('Saving image {} to files'.format(file_name))
+
+        try:
+            file_hash = hashlib.sha1(body_content.encode()).hexdigest()
+            files = parsed_mail[PROC_EMAIL_JSON_FILES]
+            files.append({'file_name': file_name, 'file_path': file_path, 'file_hash': file_hash})
+        except Exception as e:
+            _debug_print("Error occurred while adding file {} to files. Error Details: {}".format(file_name, e))
+        return
```

Comment:
This check is to ensure we don't include base64 images in the Email artifact. This is to remove noise for better analysis. This should be fine, as we will pull the images into a new Vault artifact, so it will be scanned separately in the end.